### DISCUSSION TITLE
Seeking help in cross testing this CFDP implementation

### DISCUSSION THREAD

**Discussion by ShayanMajumder** [nasa/cFS#936] [Q&A]
I am reaching out to seek your help in cross-testing our CFDP implementation for the Python version built by us at LibreCube. As a beginner in C++ and new to the cFS software, I would appreciate any assistance you can provide in setting up a client or server or any tutorial for beginners that you may have available.

Our team has been working on developing a Python-based CFDP implementation, and we would like to ensure that it is compatible with the cFS software. We believe that your expertise in this area would be invaluable in helping us to achieve this goal.

Specifically, we are seeking assistance in the following areas:

1. Setting up a client and server for CFDP using the cFS software
2. Cross-testing our Python-based CFDP implementation with the cFS software
3. Any tutorial or guidance that you can provide for beginners in using the cFS software

**Comment by chillfig** [comment]
@ShayanMajumder 
3. Training can be found at https://ntrs.nasa.gov/citations/20210022378
